README.md

```diff
@@ -93,36 +93,31 @@ from tempo.serve.loader import save
 save(classifier)
 ```
 
-Deploy to docker.
+Deploy locally to docker.
 
 ```
-from tempo.seldon.docker import SeldonDockerRuntime
-docker_runtime = SeldonDockerRuntime()
-docker_runtime.deploy(classifier)
-docker_runtime.wait_ready(classifier)
+from tempo import deploy
+rm = deploy(classifier)
```

Comment:
What does rm stand for? Maybe just `model`?

Reply:
remote model. But could change.

Reply:
ah that makes sense - maybe `remote_model` would be good